### ISSUE TITLE
Choose automatic/random bibliogram instance

### ISSUE THREAD

**Issue by lucanomax** [SimonBrazell/privacy-redirect#258]
Almost all bibliogram instances are blocked (error 502) and you have to change manually each instance until finding something that works. What about an automatic instance picker, where It picks a random instance every time you click on a link? So you don't need to change manually every time in the settings.

**Comment by xanoni** [comment]
> this feature does exist tho
> 
> Advanced => Bibliogram => Bibliogram random instance pool (comma-separated)

You're right. Didn't realize because I stopped using it ... the public instances are all (?) broken.
